src/components/project/Project.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
```suggestion
import React from "react";
```
Oops, maybe this is a missing pillow. Seems its not required.